opendbc/car/hyundai/values.py

```diff
@@ -413,7 +412,7 @@ class CAR(Platforms):
     flags=HyundaiFlags.MANDO_RADAR | HyundaiFlags.EV,
   )
   KIA_NIRO_EV_2ND_GEN = HyundaiCanFDPlatformConfig(
-    [HyundaiCarDocs("Kia Niro EV 2023-24", "All", car_parts=CarParts.common([CarHarness.hyundai_a]))],
+    [HyundaiCarDocs("Kia Niro EV 2023-25", "All", car_parts=CarParts.common([CarHarness.hyundai_a]))],
```

Comment:
maybe something like this
```suggestion
    [
      HyundaiCarDocs("Kia Niro EV (without HDA II) 2023-25", "Highway Driving Assist", car_parts=CarParts.common([CarHarness.hyundai_a])),
      HyundaiCarDocs("Kia Niro EV (with HDA II) 2023-25", "Highway Driving Assist II", car_parts=CarParts.common([CarHarness.hyundai_r])),
    ],
```